TheraLang.Web/ClientApp/src/app/core/http/project/project.service.ts

```diff
@@ -64,6 +66,13 @@ export class ProjectService {
   }
 
   addProject(project: Project) {
+    let token: string = localStorage.getItem("jwt");
+    if (token && !this.jwtHelper.isTokenExpired(token)) {
```

Comment:
maybe better use isAuthenticated() method from UserService?

Reply:
i think it's way better than hooking up a whole service because we use a token to extract the user id and immediately check it for validity

Reply:
Move getting userid method to user ( or auth ) service

Reply:
You are right. Done.